<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-FJKP7PT" failures="1" tests="2" name="Test(failed)" time="8.836" errors="0" timestamp="2024-12-15T17:22:50 IST">
  <testcase classname="com.pi.utility.Example" name="methodtwo" time="0.033">
    <failure type="java.lang.AssertionError" message="expected [https://www.amao] but found [https://www.amazon.in/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.amao] but found [https://www.amazon.in/]
at com.pi.utility.Example.methodtwo(Example.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- methodtwo -->
  <testcase classname="com.pi.utility.Example" name="methodthree" time="0.0">
    <skipped/>
  </testcase> <!-- methodthree -->
  <testcase classname="com.pi.utility.Example" name="methodone" time="0.0">
    <ignored/>
  </testcase> <!-- methodone -->
</testsuite> <!-- Test(failed) -->
